app/views/expenses/_form.html.erb

```diff
@@ -0,0 +1,38 @@
+<%= form_with url: category_expenses_path(params[:category_id]), scope: :expense,  class:"w-full" do |form| %>
+  <% if expense.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(expense.errors.count, "error") %> prohibited this expense from being saved:</h2>
+
+      <ul>
+        <% expense.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div>
+    <%= form.text_field :name, class: "w-full py-1 pl-1 rounded-sm mb-5", placeholder: 'Name' %>
+  </div>
+
+  <div>
+    <%= form.text_field :amount, class: "w-full py-1 pl-1 rounded-sm mb-5", placeholder: 'Amount', inputmode:"decimal" %>
+  </div>
+
```

Comment:
- [ ] I would recommend adding the feature to select the `categories` here in the form as per requirements. Please update this.

![Screenshot from 2022-04-30 00-17-18](https://user-images.githubusercontent.com/22434660/166007619-724ba4ad-f7a5-4407-8a23-c33dfc19da61.png)